documentation/more_information/cheat_sheets.qmd

```diff
@@ -6,4 +6,4 @@ This page will contain cheat sheets for the most commonly commands and concepts
 
 ## Viash
 
-[![Cheat sheet for developing modular pipeline components with Viash, including a sample Viash component (left) and common commands used throughout the various stages of a development cycle (right).](../images/viash_figure_3.png){#fig-viash-cheat-sheet}](../images/viash_figure_3.pdf)
\ No newline at end of file
+[Cheat sheet](https://viash.io/cheatsheet/viash_cheatsheet_v1_1.pdf) for developing modular pipeline components with viash, including a sample viash component (left) and common commands used throughout the various stages of a development cycle (right).
```

Comment:
Please include a thumbnail image for the cheat sheet. If you don't want to host the thumbnail at openproblems.bio (which is currently the case), ask @Grifs to add thumbnail pngs to viash.io.

```suggestion
[![Cheat sheet for developing modular pipeline components with Viash, including a sample Viash component (left) and common commands used throughout the various stages of a development cycle (right).](../images/viash_figure_3.png){#fig-viash-cheat-sheet}](https://viash.io/cheatsheet/viash_cheatsheet_v1_1.pdf)
```